backend/worker.py

```diff
@@ -233,10 +235,12 @@ def post(self):
 
         @ndb.transactional(xg=True, retries=10)
         def apply_remove_operation(remove_hierarchy, institution, user):
-            institution.handle_hierarchy_removal(remove_hierarchy, user)
+            callback_method = institution.handle_hierarchy_removal(remove_hierarchy, user)
```

Comment:
This method return does not seem to be a callback, would it be better to leave that name just 'returned_method'.